tests/ai_hub/model_catalog/plugin_arch/test_unified_catalog_server.py

```
@@ -28,41 +28,55 @@ def test_catalog_pod_initializes_plugins(
+
+    def test_healthz(
+        self,
+        catalog_base_url: str,
+        model_registry_rest_headers: dict[str, str],
+    ):
+        """Given the catalog server is running
+        When querying the healthz endpoint
+        Then the server reports ok status
+        """
+        response = requests.get(
+            f"{catalog_base_url}/healthz", headers=model_registry_rest_headers, verify=False, timeout=30
```

Ruff (0.15.15)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)